src/app/defaults.ts

```diff
@@ -3,6 +3,8 @@ import * as chalk from "chalk";
 export const configurationErrorEventName: string = "configuration-error-generator-office";
 export const copyFilesErrorEventName: string = "copy-files-error-generator-office";
 export const installDependenciesErrorEventName = "install-dependencies-error-generator-office";
+export const networkShareSideloadingSteps: string = "https://docs.microsoft.com/en-us/office/dev/add-ins/testing/create-a-network-shared-folder-catalog-for-task-pane-and-content-add-ins";
+export const outlookSideloadingSteps = "https://docs.microsoft.com/en-us/office/dev/add-ins/outlook/sideload-outlook-add-ins-for-testing";
```

Comment:
Can you remove `en-us` from the links so they're locale-agnostic?

Reply:
Ah, good catch.  I will have to get this in a separate PR, as this one has already been merged and a new version of Yo Office has been published